tools/devtool

```diff
@@ -757,113 +740,25 @@ cmd_build() {
         shift
     done
 
-    target="$TARGET_PREFIX${libc}"
-
     # Check prerequisites
     ensure_devctr
     ensure_build_dir
 
-    say "Starting build ($profile, $libc) ..."
-
-    # Cargo uses the debug profile by default. If we're building the release
-    # binaries, we need to pass an extra argument to cargo.
-    cargo_args=("$@")
-
-    # Add the default target if we did not get that argument in the build command.
-    add_default_target=true
-    for flag in "${@}"; do
-        if [[ "$flag" == "--" ]]; then
-            break
-        elif [[ "$flag" == "--target" || "$flag" =~ --target=.* ]]; then
-            add_default_target=false
-        fi
-    done
-
-    if [ "$add_default_target" = true ]; then
-        cargo_args+=(--target "$target")
-    fi
-
-    [ $profile = "release" ] && cargo_args+=("--release")
-
     # Map the public and private keys to the guest if they are specified.
     [ ! -z "$host_pub_key_path" ] && [ ! -z "$host_priv_key_path" ] &&
         extra_args="--volume $host_pub_key_path:$PUB_KEY_PATH:z \
                     --volume $host_priv_key_path:$PRIV_KEY_PATH:z"
 
-    # Artificially trigger a re-run of the build script,
-    # to make sure that `firecracker --version` reports the latest changes.
-    touch "$FC_ROOT_DIR/build.rs"
-
     # Run the cargo build process inside the container.
     # We don't need any special privileges for the build phase, so we run the
     # container as the current user/group.
-
-    # Build seccompiler-bin.
-    run_devctr \
-        --user "$(id -u):$(id -g)" \
-        --workdir "$CTR_FC_ROOT_DIR" \
-        ${extra_args} \
-        -- \
-        cargo build -p seccompiler --bin seccompiler-bin \
-            --target-dir "$CTR_CARGO_SECCOMPILER_TARGET_DIR" \
-            "${cargo_args[@]}"
-    ret=$?
-
-    [ $ret -ne 0 ] && return $ret
-
-    # Build rebase-snap.
     run_devctr \
         --user "$(id -u):$(id -g)" \
         --workdir "$CTR_FC_ROOT_DIR" \
         ${extra_args} \
         -- \
-        cargo build -p rebase-snap \
-            --target-dir "$CTR_CARGO_REBASE_SNAP_TARGET_DIR" \
-            "${cargo_args[@]}"
+        ./tools/release.sh --libc $libc --profile $profile
```

Comment:
Nit: It doesn't matter with the current implementation, but devtool build sets `$profile` to either `debug` or `release`, while devtool/release uses `dev` instead of `debug`.

Reply:
Good catch. Would this work to keep it backwards compatible?

```sh
            "--debug")      { profile="dev";        } ;;
```

Reply:
I think so!

Reply:
Ah I remember... that was to deal with Cargo's ambiguity of `dev` profile mapping to `debug` target directory.